website/docs/fb/configerator-config-source.md

```diff
@@ -0,0 +1,52 @@
+---
+id: fbcode-configerator-config-source
+title: Configerator Config Source Plugin
+---
+
+The ConfigeratorConfigSource plugin makes it possible for Hydra applications to consume a domain of configs from configerator.
+
+### Dependency
+
+Add the following to your `TARGET` file
+```commandline
+//fair_infra/fbcode_hydra_plugins/configerator_config_source:configerator_config_source
+```
+
+### Usage
+
+1. Since `configerator` does not have a client for listing configs under a path, you need to group your configs under
+a `domain` for Hydra to access. 
+   1. Matching the name of your domain and directory of configs is necessary for the plugin to extract information on the full config names, due to limitations from Configerator APIs
```

Comment:
Tell me what I need to do first.
The why can come later.

```
The Configerator Config Source plugin requires that you place your Configerator configs under a configerator domain (link).

Example for setting up a domain.

::: info
A domain enables listing of configerator config files which is a required functionality for this plugin to function correctly.
```

Reply:
fixed.

also included a screen shot in the PR description to show how it looks.